### PR TITLE
Update custom-types.json

### PR DESCRIPTION
**Pull request by robertsky**
hjson.org domain squatted. https://github.com/hjson/hjson/issues/96

### CODE REVIEW

**Comment by wesleytodd** [comment]
Can you give some more details on this? Are you a maintainer of hjson? I see the websites for these two links line up, but I would prefer a bit more detail before we merge. Maybe an approval from an hjson maintainer?

**Review by blakeembrey — APPROVED** [review]

**Comment by blakeembrey** [comment]
@wesleytodd The new domain is referenced in the README and GitHub repo: https://github.com/hjson/hjson. And hjson.org is indeed spam.

**Review by UlisesGascon — APPROVED** [review]

**Comment by robertsky** [comment]
> Can you give some more details on this? Are you a maintainer of hjson? I see the websites for these two links line up, but I would prefer a bit more detail before we merge. Maybe an approval from an hjson maintainer?

not a maintainer. just a passerby. I contribute more on Wikipedia nowadays, and such domain squatting to promote gambling spam is becoming [an issue there](https://en.wikipedia.org/wiki/Wikipedia:Long-term_abuse/Judi) as well. Just trying to do my part to reduce the visibility of such squatted domains.

The domain registered here may have lapsed this year resulting in the domain takeover, but from the issue I linked in my opening comment shows that the maintainer(s) there have largely given up on the domain way earlier.

**Review by IamLizu — APPROVED** [review]

**Review by bjohansebas — APPROVED** [review]

**Review by wesleytodd — APPROVED** [review]

**Pull request merged**
